src/app/shared/form/button/button.component.html

```diff
@@ -0,0 +1,5 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
I think we should move font-awesome to global/vendor styles

Reply:
get font-awesome from npm repository and add package to dependencies